docs/docs/cmd/aad/o365group/o365group-add.md

```diff
@@ -36,6 +36,7 @@ m365 aad o365group add [options]
 ## Remarks
 
 When specifying the path to the logo image you can use both relative and absolute paths. Note, that ~ in the path, will not be resolved and will most likely result in an error.
+The Owners and Members should be valid Microsoft 365 Users. If an invalid user is provided in the comma-separated list, the command operation will fail.
```

Comment:
Let's skip `The Owners and Members should be valid Microsoft 365 Users.` as all input we expect should be valid. Let's keep the explanation of what's going to happen if the input isn't valid and what failure actually mean (eg. the group won't be created).